pip/pip-274.md

```diff
@@ -0,0 +1,65 @@
+# Motivation
+
+The metrics are all started with `pulsar_`, so that both users and operators can quickly find the metrics of the entire system through this prefix. However, due to some other reasons, 
+it was found that `topic_load_times` was missing the prefix, so want to get it right.
+
+# Goals
+
+## In Scope
+
+### Proposed changes
```

Comment:
Could you move this to High Level Design section? This is not a goal

Reply:
yes, thanks for reviewing. Addressed